datahub-web-react/src/app/lineage/__tests__/LineageEntityView.test.tsx

```diff
@@ -32,7 +32,7 @@ describe('LineageEntityView', () => {
     it('should not render a divider if there is no platform name', () => {
         const datasetNoPlatformName = {
             ...dataset1,
-            platform: { ...dataset1.platform, properties: { displayName: '' } },
+            platform: { ...dataset1.platform, name: '', properties: { displayName: '' } },
```

Comment:
Why empty string ?

Reply:
Before the changes in LinageEntityView.tsx only the displayName of the platform was used, therefore the name of the platform must also be set to an empty string (or undefined) for this test to be successful. If I think about it again, in theory we could also remove this test completely, because the platform name should never be empty, however as the platform in GenericEntityProperties is optional it is possible that getPlatformName returns undefined...in this case the test has somehow the right to exist...